intel_extension_for_transformers/neural_chat/docker/Dockerfile

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+#
+# THIS IS A GENERATED DOCKERFILE.
+#
+# This file was assembled from multiple pieces, whose use is documented
+# throughout. Please refer to the TensorFlow dockerfiles documentation
+# for more information.
+#
+# ============================================================================
+# How to build: 
+#   docker build ./ -f Dockerfile -t chatbot_finetune:latest
+# If you need to use proxy, please use the following command
+#   docker build ./ --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${http_proxy} -f Dockerfile -t chatbot_finetune:latest
+
+## SPR environment
+ARG UBUNTU_VER=22.04
+FROM ubuntu:${UBUNTU_VER} as cpu
+
+ARG ITREX_VER=main
+ARG REPO=https://github.com/intel/intel-extension-for-transformers.git
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+# Install system dependencies
+SHELL ["/bin/bash", "--login", "-c"]
+RUN apt-get update \
+    && apt-get install -y build-essential \
+    && apt-get install -y wget numactl git \
+    && apt-get install -y openssh-server \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download ITREX code
+RUN git clone --single-branch --branch=${ITREX_VER} ${REPO} intel-extension-for-transformers

```

Comment:
can we use itrex binary?

Reply:
No, we've recently made several fixes. We can use itrex binary after r1.2 release.